test/unit/test_scenario.py

```diff
@@ -64,12 +64,14 @@ def test_ephemeral_directory_property(_instance):
 
 def test_check_sequence_property(_instance):
     x = [
+        'cleanup',
```

Comment:
I had hesitations on this one myself. The cleanup playbook -should- be idempotent which means it would only clean up resources that it created but that may be tricky. I wanted to stick it directly before every destroy action to ensure a cleanup happened before -every- destroy.

I feel like documenting this is probably the right call.